src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
-
-If the service provider fails to deliver on that promise, some kind of penalty is usually applied, like a partial or full refund, or monetary compensation.
-
-### Service-level Indicator
-
-A **service-level indicator**, or **SLI** is the metric we use to measure whether a service meets the [service-level objective (SLO)](#service-level-objective). While doing performance monitoring this could, for instance, be the number of successful requests against the service during a specified period.
-
-### Service-level Objective
-
-A **service-level objective**, or **SLO** is an actual target, either internal or part of the [service-level agreement (SLA)](#service-level-agreement), for the availability of the service. This is often expressed as a percentage (99,2%, for instance). If the service meets or exceeds this target, it's deemed stable.
-
-### Smoke test
-
-A **smoke test** is a type of test used to verify that the system under test can handle a minimal amount of load without any issues. It’s commonly used as a first step, to ensure that everything works as intended under optimal conditions, before advancing to any of the other performance test types. See [Smoke Testing](/test-types/smoke-testing).
-
-### Soak test
-
-A **soak test** is a type of test used to uncover performance and reliability issues stemming from a system being under pressure for an extended period. See [Soak Testing](/test-types/soak-testing).
-
-### Stability
-
-**Stability** is a trait used to describe a system under test’s ability to withstand failures and erroneous behavior under normal usage.
-
-### Stress test
+<dt id="k6-cloud">k6 Cloud</dt>
+<dd>The proper name for the entire cloud product, comprising both k6 Cloud Execution and k6 Cloud Test Results.
+<p>Read more: [k6 Cloud docs](https://k6.io/docs/cloud)</p></dd>
 
-A **stress test** is a type of test used to identify the limits of what the system under test is capable of handling in terms of load. See [Stress Testing](/test-types/stress-testing).
+<dt id="k6-options">k6 options</dt>
+<dd>Values that configure a k6 test run. You can set options with command-line flags, environment variables, and in the script.
+<p>Read more: [k6 Options](/using-ky/k6-options)</p></dd>
 
-### System under test
+<dt id="load-test">Load test</dt>
+<dd>A test that assesses the performance of the system under test in terms of concurrent users or requests per second.
+<p>Read more: [Load Testing](/test-types/load-testing)</p></dd>
 
-**System under test** refers to the actual piece of software that we're currently testing. This could be an API, a website, infrastructure, or any combination of these.
+<dt id="load-zone">Load zone</dt>
+<dd>The geographical instance from which a test runs.
+<p>Read more: [Private load zones](/cloud/cloud-faq/private-load-zones/)</p></dd>
```

Comment:
Add link to https://k6.io/docs/cloud/creating-and-running-a-test/cloud-tests-from-the-cli/#load-zones